app/src/main/java/com/karumi/domain/model/Errors.kt

```diff
@@ -0,0 +1,7 @@
+package com.karumi.domain.model
+
+sealed class DomainError
+class NotInternetDomainError : DomainError()
```

Comment:
`object NotInternetDomainError` and `object AuthDomainError` instead if they are not holding any information.

Reply:
ok